packages/insomnia-app/app/main.development.ts

```diff
@@ -39,6 +39,12 @@ if (!isDevelopment()) {
 global.window = global.window || undefined;
 // When the app is first launched
 app.on('ready', async () => {
+  // There's no option that prevents Electron from fetching spellcheck dictionaries from Chromium's CDN and passing a non-resolving URL is the only known way to prevent it from fetching.
+  // see: https://github.com/electron/electron/issues/22995
+  // On macOS the OS spellchecker is used and therefore we do not download any dictionary files.
+  // This API is a no-op on macOS.
+  session.defaultSession.setSpellCheckerDictionaryDownloadURL('https://00.00/');
+
```

Comment:
I like how this `app.on('ready')` callback has mostly a series of abstracted steps where the functions called are themselves somewhat simple.  E.g. `_createModelInstances`, `errorHandling.init()`, and `windowUtils.init()` are all, when you dig in, just 2 line functions.  I'd suggest we make a private function in this module called something like, oh, I donno, this:

```ts
/**
 * There's no option that prevents Electron from fetching spellcheck dictionaries from Chromium's CDN and passing a non-resolving URL is the only known way to prevent it from fetching.
 * see: https://github.com/electron/electron/issues/22995
 * On macOS the OS spellchecker is used and therefore we do not download any dictionary files.
 * This API is a no-op on macOS.
 */
const disableSpellcheckerDownload = () => {
  session.defaultSession.setSpellCheckerDictionaryDownloadURL('https://00.00/');
};
```

That way we can just call it here:
```suggestion
  disableSpellcheckerDownload();
```

Reply:
I agree `disableSpellcheckerDownload` has clearer intent.

Reply:
I wonder if it's possible to unit test this _somehow_ to make sure we never lose this behavior (as we need to guarantee it), though I agree with pulling out a helper function as well.

Reply:
Good one! Moved it to electron-helpers.
I'll try to make a PR to test this, but couldn't think of a nice way with the current spectron apis.